.gradle/caches/modules-2/files-2.1/com.liferay/com.liferay.calendar.api/2.0.1/d261e85d671ef83551446c0e8775c7419d4c0129/com.liferay.calendar.api-2.0.1.pom

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Marcellus told me that it is not necessary anymore to commit the cache! Good news, right :sunglasses: I will take this commit out, then